CMakeLists.txt

```diff
@@ -4,7 +4,12 @@ include(ExternalProject)
 
 if (NOT "${CMAKE_PROJECT_NAME}" STREQUAL "tilck")
 
-   project(tfblib C)
+   project(tfblib
+      LANGUAGES C
+      HOMEPAGE_URL https://github.com/vvaltchev/tfblib
+      DESCRIPTION "A Tiny Linux Framebuffer Library"
+      VERSION "0.1"
```

Comment:
There aren't any tagged releases of tfblib yet, this can be changed to be generated from a git tag / the current date, but using a version number is a lot easier.

Reply:
Yeah, I agree: using a version number is better. Starting with ver 0.1 is perfect.